src/main/java/seedu/address/model/person/Client.java

```diff
@@ -44,6 +44,19 @@ public Products getProducts() {
         return this.products;
     }
 
+    /**
+     * Returns the products preferred by the client.
+     *
+     * @return The products preferred by the client as a String.
+     */
+    public String getProductsAsString() {
+        String result = String.join(", ", products.getProducts());
+        if (result.isEmpty()) {
+            return result;
+        }
+        return "\"" + result + "\"";
```

Comment:
Why we encapsulate the list of products using `"` and `"`. I personally think that in the exported csv file should not have this as it seems less practical to a user to copy over and use the data.

Reply:
How CSV file store data is that it will read the string and it will cut the content tp then new cell when comma (,)is found. Therefore, to storetext that contains "," (for tags, skills, prods fields) we need to add " in front and an the end for the csv to know that these commas are for content not to seperate